src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2037,6 +2054,12 @@ def set_up_addon_profiles(self, mc: ManagedCluster) -> ManagedCluster:
             addon_profiles[
                 CONST_GITOPS_ADDON_NAME
             ] = self.build_gitops_addon_profile()
+
+        if self.context.get_enable_cluster_snapshot():
+            supported = ["coredns", "overlay-upgrade-data", "kube-proxy", "tunnelfront", "metrics-server", "csi-azurefile-node", "csi-azuredisk-node"]
```

Comment:
how people manage the list?

Reply:
They are core addons. Right now, we only snapshot core addons

Reply:
Maybe it's better to put this list in `_consts.py`?